### PR TITLE
Picsure 67 2.0 docker

### PR DESCRIPTION
**Pull request by YuAvillachLab**
The picsure 2 Dockerfile is the most important piece

### CODE REVIEW

File: deployments/picsure-2/env.env [hunk 1]
```diff
@@ -0,0 +1,34 @@
+### for all environments ####
```
Comment (calypsomatic) [inline deployments/picsure-2/env.env]: Is there a difference between .env and env.env?

Comment (YuAvillachLab) [reply deployments/picsure-2/env.env]: Good question, I mostly worked on Dockerfile and quickstart, I will delete all these parts, and work on it based on IRCT

Comment (recursethis) [reply deployments/picsure-2/env.env]: `env.env` is deprecated. I used it so as never to commit a `.env` file, but that caused problems with simple deployments with development and production. Basically in your deployments repo you should **now** have an existing `.env` and remove any `env.env` files.

**Comment by recursethis** [comment]
I'll review the commit later this evening

**Pull request merged**